data/dineshlalam15.json

```
@@ -13,5 +13,15 @@
       "url": "https://github.com/dineshlalam15",
       "icon": "FaGithub"
     }
```

There is another missing comma `,`

```suggestion
    },
```